test.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+require '../functions.php';
+
+use PHPUnit\Framework\TestCase;
+
+class Functions extends TestCase
+{
+	public function testSuccessDisplayBooks()
+	{
+		$example = ['authorFirstName' => 'authorFirstName', 
+                    'authorSecondName' => 'authorSecondName', 
```

Comment:
indentation needs looking at here

Reply:
I won't look at test until you have passed your end!

Reply:
indent seems fine here but i could have got wrong end of stick

Reply:
not sure what would be correct indentation here